app/jobs/send_password_reset_email_job.rb

```diff
@@ -0,0 +1,9 @@
+class SendPasswordResetEmailJob < ActiveJob::Base
+
+  queue_as :default
```

Comment:
no need to specify default explicitly

Reply:
this

Reply:
Please do make sure to fix all previous issues before you re-submit the exercise.